src/folders_and_files/files.md

```diff
@@ -3,12 +3,12 @@
 BIDS has a standardized way of naming files that tries to implement the
 following principles:
 
--   Do not white spaces in file names
+-   Do not includwe white spaces in file names
```

Comment:
There is a typo in this sentence (includwe should not have a w)

Reply:
:see_no_evil: 

will fix it

Reply:
fixed by #279 
thanks @dorahermes

Reply:
Oops 🙈 my bad!